misc-tools/dj_make_chroot.in

```diff
@@ -35,15 +35,15 @@ DEBIAN_ARCHLIST="amd64,arm64,armel,armhf,i386,mips,mips64el,mipsel,ppc64el,s390x
 UBUNTU_ARCHLIST="amd64,arm64,armhf,i386,powerpc,ppc64el,s390x"
 
 # If host system is Debian or Ubuntu, use its release and architecture by default
-if [ "x$(lsb_release -i -s || true)" = 'xDebian' ] || \
-   [ "x$(lsb_release -i -s || true)" = 'xUbuntu' ]; then
+if [ "$(lsb_release -i -s || true)" = 'Debian' ] || \
+   [ "$(lsb_release -i -s || true)" = 'Ubuntu' ]; then
```

Comment:
Was shellcheck complaining about this? I realize this is maybe (over)cautious escaping, but doesn't hurt as far as I'm aware.

Reply:
I think the removal of `x` is because of: https://github-wiki-see.page/m/koalaman/shellcheck/wiki/SC2268

Reply:
Yes, it was exactly that lint.